framework/components/ema-separator/ema-separator.twig

```diff
@@ -0,0 +1,37 @@
+{#
+  Accepts:
+    - title (string): title of the panel (default: '')
+    - icon (string): name of the ECL-icons class to be used in the heading
+    - extra_classes (string): additional classes to add to the component
+    - extra_attributes (array of { name: '', value: '' })
+#}
+
+{# Internal properties #}
+{% set _css_class = 'ema-separator' %}
+{% set _extra_attributes = '' %}
+{% set _title = title|default('') %}
+{% set _icon = icon ~ ' ecl-icon'|default('') %}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print result #}
+<div class="{{ _css_class }}"{{ _extra_attributes }}>
+  <div class="ema-separator__heading">
+    <h2 class="ema-separator__heading--title">
```

Comment:
I'd rather use a h3 here (these separator will come inside a page which already contains a h2 as title).
Also, please add the ecl classes for h3 (ecl-heading ecl-heading--h3). Part of them could be overwritten if needed, but at least that will follow global look and feel of typography.